### ISSUE TITLE
Add DebuggerVisualisers to nuget package

### ISSUE THREAD

**Issue by geometrikal** [emgucv/emgucv#159]
Thanks so much for putting emgucv on nuget

Is it possible to put the debugger visualiser dll up there as well?

**Comment by geometrikal** [comment]
Yes I know, but it would save another download - use could just copy from the packages directory. (Also, it doesn't have to be in the Visual Studio folder, it can be put in \Users\USERNAME\Documents\Visual Studio 2017\Visualizers)

**Comment by emgucv** [comment]
The Emgu CV debugger visualizer is available in the Visual Studio Marketplace and can be installed directly from within Visual Studio without a separate download or admin rights. Closing now.
